style.css

```diff
@@ -509,6 +542,54 @@ textarea {
   text-align: center;
 }
 
+.popup-container {
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  backdrop-filter: brightness(0.5);
+}
+
+.container-popup {
+  padding: 16px;
+  background-color: #fff;
+  border-radius: 26px;
+  margin: 15% 5%;
+  display: flex;
+  flex-direction: column;
+}
+
+.underline {
+  border-bottom: 1px solid black;
+  margin-bottom: 30px;
+  margin-top: 23px;
+}
+
+.pop-button {
+  display: flex;
+  gap: 20px;
+  margin-bottom: 20px;
+}
+
+.live {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
+.see-project a {
+  text-decoration: none;
+}
```

Comment:
- [ ] Kindly correct the button size, add the given styles for, hover, active, and disabled, and the color of the bar above the buttons to be similar to the Figma template 👇🏽 

| Figma | Your design |
|---|---|
| ![image](https://github.com/alizaamir123/mobile-version/assets/58642949/7c343acc-87a7-4990-8e73-1d8fea5c4a89) | ![image](https://github.com/alizaamir123/mobile-version/assets/58642949/055c1a94-ca51-418f-b04b-ef6113ba77ff) |